app/models/transcript.rb

```diff
@@ -10,9 +17,12 @@ class Transcript < ApplicationRecord
 
   validates :original_url, presence: true
 
-  validates :mime_type, inclusion: {in: %w[text/plain application/json]}, if: :status_complete?, allow_blank: true
+  validates :mime_type, inclusion: {in: MIME_TYPES.values}, if: :status_complete?, allow_blank: true
+
+  validates :format, presence: true
 
   enum :status, [:started, :created, :processing, :complete, :error, :retrying, :cancelled, :invalid], prefix: true
+  enum :format, MIME_TYPES, prefix: true, default: :json
```

Comment:
Hmm... I see where you're coming from here (this is nice and DRY code).

But what Rails is actually doing is storing the string `text/html` in the database.  And translating that to `html` in the model.

I can imagine that causing problems down the road (imagine we add a 2nd `format` that _just happens_ to also have the mime-type `text/html`).

So I think we want to store the literal string `html` in the database here?  So the hash we pass in should be `{html: "html"}` ... not `{html: "text/html"}` here.